src/gdal/rasterband.jl

```diff
@@ -257,8 +257,8 @@ _getrasterscale(hBand::GDALRasterBandH, pbSuccess::Ptr{Cint}) =
     GDALGetRasterScale(hBand, pbSuccess)::Cdouble
 
 function getrasterscale(rasterband::GDALRasterBandH)
-    success = Ref{Cint}()
```

Comment:
This didn't work for me anymore. But I don't know about C interop.  This would probably need changing elsewhere too.